cloudify_aws/ec2/elasticloadbalancer.py

```diff
@@ -54,14 +54,15 @@ def disassociate(args=None, **_):
 class ElbInstanceConnection(AwsBaseRelationship):
 
     def __init__(self, client=None):
-        super(ElbInstanceConnection, self).__init__(client=client)
+        self.client = client or connection.ELBConnectionClient().client()
+        super(ElbInstanceConnection, self).__init__(client=self.client)
         self.not_found_error = 'InvalidInstanceID.NotFound.'
         self.resource_id = None
-        self.client = connection.ELBConnectionClient().client()
         self.source_get_all_handler = {
             'function': self.client.get_all_load_balancers,
-            'argument':
+            'argument': (
                 '{0}_names'.format(constants.ELB['AWS_RESOURCE_TYPE'])
+            )
```

Comment:
Fixes flake8 complaint- not actually relevant to this PR, but it was failing flake8 for me.